src/pages/login/page.js

```diff
@@ -1,8 +1,14 @@
 import React from "react";
+import { browserHistory } from "react-router";
 import styles from "./style.css";
 
 
 export default class LoginPage extends React.Component {
+  
+  signUp(){
```

Comment:
@pheuter Done. 
@paulyoung +1. I think it's a good idea, especially when multiple contributors are involved.